### PR TITLE
Added in item content and updated model

### PR DESCRIPTION
**Pull request by lawsonbh**
Added in seed data for Item after updating the model. Included a choice list for item type and renamed the fields for clarity and consistency with the other models. I also wired Item into admin.

### CODE REVIEW

File: bodybank/character_generator/admin.py [hunk 1]
```diff
@@ -26,11 +26,15 @@ class CareerAdmin(admin.ModelAdmin):
 class Career2SkillsAdmin(admin.ModelAdmin):
     pass
 
+class ItemAdmin(admin.ModelAdmin):
+    pass
```
Comment (bbelderbos) [inline bodybank/character_generator/admin.py]: I recommend making the admin look a bit nicer, example from IronScribe:

```
@admin.register(Profile)
class ProfileAdmin(admin.ModelAdmin):
    list_display = (
        "user",
        "tier",
        "company",
        "credits",
        "can_spend_credits",
        "can_edit_transcriptions",
    )
    search_fields = ("user__username", "tier__name", "company_name")
    list_filter = ("tier", "company")
```

**Review by bbelderbos — APPROVED** [review]
ok

**Comment by lawsonbh** [comment]
After applying autoformatting on master, did my best to merge those changes in - interested in your feedback @bbelderbos if that did that in a clean enough way / how you would have approached bring that in from master.

Also - really like decorator you suggested and adding in those details to the Item admin class. Really provides some great clarity in the admin pane.

Almost went ahead to do it for the other admin classes but decided enough was going on in this PR already :raccoon:

File: bodybank/character_generator/admin.py [hunk 2]
```diff
@@ -1,36 +1,59 @@
...
     pass
 
+
 class Career2SkillsAdmin(admin.ModelAdmin):
     pass
 
+
+@admin.register(Item)
+class ItemAdmin(admin.ModelAdmin):
+    list_display = ("name", "itype", "complexity_gp", "desc")
```
Comment (bbelderbos) [inline bodybank/character_generator/admin.py:49]: More cool things you can do, but list, search and filter are the most common ones:
https://docs.djangoproject.com/en/4.1/ref/contrib/admin/

For bookmark / later reading: 
https://books.agiliq.com/projects/django-admin-cookbook/en/latest/
-> some really cool tricks

**Review by bbelderbos — COMMENTED** [review]
Yes I would probably have used one specific formatting branch and merge that in, similar to what you did. Not sure though why I saw several PRs for this?

**Pull request merged**